Automattic/wp-calypso
# Ecommerce Tailored Flow: Adding 'Woocommerce powered' logo

#### Proposed Changes

* Per request in https://github.com/Automattic/wp-calypso/issues/71244, this PR adds a "WooCommerce powered" label to the Intro, Profiler, and Processing screens, following existing patterns from the newsletter flow.

<img width="728" alt="image" src="https://user-images.githubusercontent.com/13437011/208769319-4bdc2af4-676e-48ec-9bb4-72e62ce00aca.png">
<img width="1228" alt="image" src="https://user-images.githubusercontent.com/13437011/208769350-349adf52-d500-4161-9233-df7cb60b0464.png">
<img width="1150" alt="image" src="https://user-images.githubusercontent.com/13437011/208769401-ad759c39-ef1e-4918-9728-378280221833.png">


#### Testing Instructions



* Check out this PR locally and navigate to `/setup/ecommerce`
* Navigate through the flow and verify the presence of the WooCommerce powered label


Related to #71244
